conf/fub_curta.config

```diff
@@ -0,0 +1,31 @@
+// Config profile metadata
+params {
+    config_profile_name = 'FUB Curta'
+    config_profile_description = 'Freie Universität Berlin HPC cluster (Curta) profile'
+    config_profile_contact = 'hpc@zedat.fu-berlin.de'
+    config_profile_url = 'https://www.fu-berlin.de/en/sites/high-performance-computing/index.html'
+    max_memory = 93.GB
+    max_time = 14.d
+}
+
+// Slurm configuration parameters
+process {
+    scratch = true
+    executor = 'slurm'
+    clusterOptions = '--qos=standard'
+    maxRetries = 2
+}
+
+// Singularity parameters
+singularity {
+    enabled = true
+    autoMounts = true
+}
+
+cleanup = true
+
+profiles {
+    debug {
+        cleanup = false
+    }
+}
```

Comment:
This is not needed anymore, it's integrated in the main pipeline nf-core template taht most pipelines should have (some configs still have it as legacy)

Reply:
I see. Thanks for the info. I've removed it then 👌🏼